src/multiplatform-apps.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Multi-platform app development
+short-title: Multi-platform apps
+description: Use the Flutter framework to build beautiful native apps for many platforms and operating systems from a single codebase.
+toc: false
+---
+
+We recommend the [Flutter framework][] for developing multi-platform native apps for mobile (iOS & Android), desktop (Windows, Linux, and macOS), and the web.
```

Comment:
```suggestion
We recommend the [Flutter framework][] for developing multi-platform native apps
for mobile (iOS & Android), desktop (Windows, Linux, and macOS), and the web.
```

Reply:
& -> and

(We only use `&` in space-constrained places like the short title and sidenav.)